samples/Mvc.Client/Extensions/HttpContextExtensions.cs

```diff
@@ -1,41 +0,0 @@
-﻿/*
- * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
- * See https://github.com/aspnet-contrib/AspNet.Security.OpenId.Providers
- * for more information concerning the license and the contributors participating to this project.
- */
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Http.Authentication;
-
-namespace Mvc.Client.Extensions
```

Comment:
Please keep this file. You can find an updated version here: https://github.com/aspnet-contrib/AspNet.Security.OpenIdConnect.Server/blob/dev/samples/Mvc/Mvc.Server/Extensions/HttpContextExtensions.cs#L12-L35